minimint-api/Cargo.toml

```diff
@@ -22,5 +22,8 @@ tbs = { path = "../crypto/tbs"}
 thiserror = "1.0.23"
 tracing ="0.1.22"
 
+[target.'cfg(target_arch="wasm32")'.dependencies]
+getrandom = { version = "0.2.0", features = [ "js" ] }
```

Comment:
I don't see this used anywhere yet, so why include it?

Reply:
we need to enable `js` feature of getrandom(to make it compile on wasm32-unknown-unknown). it is explained in [`docs`](https://docs.rs/getrandom/latest/getrandom/#indirect-dependencies)

Reply:
That can be slightly confusing, maybe add a comment that it's a transitive dependency of `rand` and needs that feature for WASM? Just want to avoid trying to patch it out later and scratching my head why nothing works anymore :laughing: